packages/blocks/src/CookieBanner/CookieBanner.test.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Text } from '@cmpsr/components';
+import { fireEvent, renderWithProviders, screen } from '@tests/renderWithProviders';
+import { CookieBanner } from './CookieBanner';
+
+describe('CookieBanner', () => {
+  afterEach(() => {
+    localStorage.clear();
+  });
+  test('should render children', () => {
+    renderWithProviders(<CookieBanner>Hello</CookieBanner>);
+    screen.getByText('Hello');
+  });
+  test('should render custom children', () => {
+    renderWithProviders(
+      <CookieBanner>
+        <Text data-testid="children-container">Hello</Text>
+      </CookieBanner>
+    );
+    screen.getByText('Hello');
+    screen.getByTestId('children-container');
+  });
+  test('should not render component after accepting all cookies', () => {
+    renderWithProviders(
+      <CookieBanner data-testid="cookie-banner" acceptAllCta="All">
+        <Text>Hello</Text>
+      </CookieBanner>
+    );
+    expect(screen.queryByTestId('cookie-banner')).not.toBe(null);
+    fireEvent.click(screen.getByText('All'));
+    expect(screen.queryByTestId('cookie-banner')).toBe(null);
```

Comment:
Nit: I'd use toBeVisible or toBeInTheDocument to improve readability

Reply:
done